src/main/java/net/sourceforge/plantuml/activitydiagram3/ftile/UGraphicInterceptorUDrawable2.java

```diff
@@ -106,15 +110,17 @@ private void drawGoto(FtileGoto ftile) {
 			return;
 		final double dx = dest.getDx() - posNow.getDx();
 		final double dy = dest.getDy() - posNow.getDy();
+		if (isDebug) {
```

Comment:
I tried with:
```java
(Boolean.parseBoolean(option.getValue(OptionKey.DEBUG))
```
But I don't know where to put:
```java
ConfigurationStore<OptionKey> option = diagram.getPreprocessingArtifact().getOption()
```

Reply:
@arnaudroques:
- Do you have any indications?

Reply:
Yes, I've made a commit to put you on the way
https://github.com/plantuml/plantuml/commit/e846b74531b9a80b820aa84dfc4b60866f5139d3